app/controllers/charge_controller.js

```diff
@@ -185,7 +185,7 @@ module.exports = {
 
     const correlationId = req.headers[CORRELATION_HEADER] || ''
     const payload = normalise.apiPayload(req, card)
-    if (res.locals.service.collectBillingAddress === false) {
+    if (res.locals.collectBillingAddress === false) {
```

Comment:
This was the line I forgot to change before. Was causing validation in connector to fail. It works now (tested with pay local), and added a test in test/controllers/charge_controller_create_test.js